ros-tutorials/lesson-0-introduction.md

```diff
@@ -1,24 +1,75 @@
 # Lesson 0 - Introduction
 
-The following set of tutorials will introduce you to Robot Operating System \(ROS\). They will provide a framework of goals and a listing of resources you may use to achieve these goals.
+This introduction will set up your system for the rest of the tutorials. It will expose you to some of the software we use: Unix command line, Docker, and Git.
 
-Before continuing, if you have not used the command line before, complete this short tutorial:
+## Command Line
 
-[Command Line Basics](https://www.vikingcodeschool.com/web-development-basics/a-command-line-crash-course)
+Before continuing, if you have never used the Unix command line (or Terminal in Mac or Command Prompt in Windows), complete [this short tutorial](https://www.vikingcodeschool.com/web-development-basics/a-command-line-crash-course).
 
-To run ROS, we will be using Docker. This is a software that allows you to run programs inside an isolated environment. This means that you do not need to download anything but the docker client, and the particular image we will be using. First, download the appropriate docker client:
+## Docker Setup
+
+To run ROS, we will be using Docker. This is a software that allows you to run programs inside an isolated environment called a container. So, you only need to download the Docker client and then run our image. Our image includes all of the little pieces of software we need, rather than installing them individually. If you're curious, learn more [here](https://www.docker.com/resources/what-container) or [here](https://docs.docker.com/get-started/).
+
+Download and install the appropriate Docker client:
 
 * [Mac](https://docs.docker.com/docker-for-mac/install/)
-* [Windows (Home)](https://docs.docker.com/toolbox/toolbox_install_windows/)
-* [Windows (Other)](https://docs.docker.com/docker-for-windows/install/)
-* Which Windows do I have? Start > type "PC Info" > note if you have Windows 10 Home, Pro, Education, etc.
+* [Windows (Home)](https://docs.docker.com/docker-for-windows/install-windows-home/)
+* [Windows (Pro, Education, etc.)](https://docs.docker.com/docker-for-windows/install/)
+
+:information_source: Which Windows do I have? *Settings > System > About > look at "Edition"*<br>
+:warning: If you have Windows Home, make sure you follow the installation instructions exactly. Instead, we recommend upgrading to Windows Education using [Duke's free student license](https://software.duke.edu), though it is not required.
```

Comment:
I don't think this warning is necessary - with WSL 2, everything is smooth. I don't want someone wasting their time upgrading their system, and that might break stuff since Windows doesn't do well with upgrades (lol I don't want someone suing us if all their data is deleted or something).

You however may want to mention updating their system to the May 2020 update, I think its Windows version 1903 or something. Then again, updating is probably as bad as upgrading, so not sure which is lesser of two evils.

Reply:
Sounds good. I haven't used WSL 2, so I was just going off what we've talked about previously. The system update is probably easier than upgrading editions through Duke just for this, so I'll cut out this part entirely.